### ISSUE TITLE
Extension causes high cpu load

### ISSUE THREAD

**Issue by MadMax888** [angular/vscode-ng-language-service#320]
- Issue Type: `Performance`
- Extension Name: `ng-template`
- Extension Version: `0.1.11`
- OS Version: `Darwin x64 18.2.0`
- VSCode version: `1.32.3`

:warning: Make sure to **attach** this file from your *home*-directory: `/Users/m.ivashchuk/Angular.ng-template-unresponsive.cpuprofile.txt` :warning:

Find more details here: https://github.com/Microsoft/vscode/wiki/Explain:-extension-causes-high-cpu-load

**Comment by kdcro101** [comment]
Same here. 
CPU skyrockets when angular project is opened.

Removal/Disabling of Angular Language Service extension fixes the problem.

**Comment by PeterStaev** [comment]
Getting the same here. It makes VS Code intellisense in TS files load extremely slow. I ended up disabling the extension. 
```
Version: 1.33.1 (user setup)
Commit: 51b0b28134d51361cf996d2f0a1c698247aeabd8
Date: 2019-04-11T08:27:14.102Z
Electron: 3.1.6
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763
```

**Comment by kyliau** [comment]
Released v0.800.0 that fixed performance regression.
Multiple beta users confirmed that language service works well with Angular v7 and v8.
Please open another issue if this is not the case.
https://github.com/angular/vscode-ng-language-service/releases/tag/v0.800.0

Related issues:
- https://github.com/angular/vscode-ng-language-service/issues/329
- https://github.com/angular/angular/issues/27904
